src/Operators/finitedifference.jl

```diff
@@ -2106,13 +2106,32 @@ function Base.Broadcast.materialize!(
     )
 end
 
+
+unwrap_ref(r::Ref) = r[]
+unwrap_ref(r) = r
+
+@inline function column_op(f::DivergenceF2C, inds...)
+    DivergenceF2C(map(bc -> column_bc(bc, inds...), f.bcs))
+end
+
+@inline function column_op(f::InterpolateC2F, inds...)
+    InterpolateC2F(map(bc -> column_bc(bc, inds...), f.bcs))
```

Comment:
These need to be implemented for all operators. Not sure if there is a nicer way to do it.

Reply:
It would be good to test if the optimizer works with these before maybe continuing.

`map -> tuplemap` might help a bit

Reply:
I feel we get some speedups and new even more indirection are added :)